src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -139,71 +181,89 @@ function NodeComponent({handleFocusTabs}: {handleFocusTabs: () => void}) {
+                                        content={copied ? 'Copied!' : 'Copy to clipboard.'}
+                                        trigger="mouseenter click"
+                                    >
+                                        <Clipboard
+                                            className="resource-action-tabs__active icon-dim-12 pointer ml-8 mr-8"
+                                            onClick={(e) => {
+                                                e.stopPropagation();
+                                                copyToClipboard(node?.name, () => setCopied(true));
                                             }}
-                                            className="fw-6 cb-5 ml-6 cursor resource-action-tabs__active"
+                                        />
+                                    </Tippy>
+                                    {getNodeDetailTabs(node.kind).map((kind, index) => {
+                                        return (
+                                            <a
+                                                key={'tab__' + index}
+                                                onClick={() => {
+                                                    if (node.kind === NodeType.Containers) {
+                                                        handleActionTabClick(node['pNode'], kind, node.name);
+                                                    } else {
+                                                        handleActionTabClick(node, kind);
+                                                    }
+                                                    handleFocusTabs();
+                                                }}
+                                                className="fw-6 cb-5 ml-6 cursor resource-action-tabs__active"
 
-                                        >
-                                            {kind}
-                                        </a>
-                                    );
-                                })}
+                                            >
+                                                {kind}
+                                            </a>
+                                        );
+                                    })}
+                                </div>
                             </div>
+                            {node.kind === NodeType.Pod && podLevelExternalLinks.length > 0 && (
+                                <NodeLevelExternalLinks
+                                    helmAppDetails={appDetails}
+                                    nodeLevelExternalLinks={podLevelExternalLinks}
+                                    podName={node.name}
+                                />
+                            )}
+                            {node.kind === NodeType.Containers && containerLevelExternalLinks.length > 0 && (
+                                <NodeLevelExternalLinks
```

Comment:
Please code is generic way so that if external link comes on any level then integration is easy. for eg : on dpeloyment level

Reply:
Sure. will try to refactor this to be more generic.

Reply:
As discussed, keeping it only for 2 kinds for now i.e. `Pod` & `Container`. Will make it more generic to support other kinds as part of enhancement.